components/DynamicForm/MyContinue.tsx

```diff
@@ -0,0 +1,30 @@
+import React, { useEffect } from 'react'
+import { QuestionText } from '../FormStyles/QuestionText'
+
+interface ContinueProps {
+  label: string
+  onMount: () => void
```

Comment:
what does onMount do?

Reply:
onMount runs when the component is mounted AKA rendered, and the function they pass in basically updates the formanswer and autoselects the "continue" answer, which isn't actually displayed to the user but still needs to be selected due to the way we handle answers